### PR TITLE
chore: provide default session credentials to pause container

### PR DESCRIPTION
**Pull request by dannyrandall**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/cli/run_local.go [hunk 1]
```diff
@@ -369,6 +369,14 @@ func (o *runLocalOpts) getTask(ctx context.Context) (orchestrator.Task, error) {
 		Containers: make(map[string]orchestrator.ContainerDefinition, len(td.ContainerDefinitions)),
 	}
 
+	if o.proxy {
```
Comment (Lou1415926) [inline internal/pkg/cli/run_local.go:372]: Why does pause need the default credentials only on `o.proxy` instead of always?

**Review by Lou1415926 — COMMENTED** [review]
The code LGTM! Just one question ⬇️

File: internal/pkg/cli/run_local.go (hunk 1 above)
Comment (dannyrandall) [reply internal/pkg/cli/run_local.go:372]: we don't do anything with them in `pause` if we aren't creating proxy connections! we (still) always inject the creds into the workload containers.

**Review by Lou1415926 — APPROVED** [review]

File: internal/pkg/cli/run_local.go [hunk 2]
```diff
@@ -442,32 +468,9 @@ func (c containerEnv) Secrets() map[string]string {
 // continer defined in the TaskDefinition. The returned map is a map of container names,
```
Comment (iamhopaul123) [inline internal/pkg/cli/run_local.go]: nit
```suggestion
// container defined in the TaskDefinition. The returned map is a map of container names,
```

File: internal/pkg/docker/orchestrator/orchestrator.go [hunk 3]
```diff
@@ -281,7 +281,8 @@ func (o *Orchestrator) containerID(name string) string {
 // Containers within a Task can talk to each other on localhost
 // and are stopped and started as a group.
 type Task struct {
-	Containers map[string]ContainerDefinition
+	Containers   map[string]ContainerDefinition
+	PauseSecrets map[string]string
```
Comment (iamhopaul123) [inline internal/pkg/docker/orchestrator/orchestrator.go:285]: Why do we need this to be a separate field as opposed to use `t.Containers["pause"].Secrets` 🤔

Comment (iamhopaul123) [reply internal/pkg/docker/orchestrator/orchestrator.go:285]: Oh nvm sorry i didn't realize we don't really have a "pause" container in `Containers`.

**Review by iamhopaul123 — APPROVED** [review]

**Pull request merged**